### PR TITLE
SUI classes are added to form elements

### PR DESCRIPTION
**Pull request by DimDev**

### CODE REVIEW

**Comment by Slavikkk** [comment]
+

File: css/custom.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.suiadmin .button-action:before {
+    display: none;
+}
```
Comment (egorbogatyrev) [inline css/custom.css]: please add an empty line

File: suiadmin_theme.theme [hunk 2]
```diff
@@ -3,7 +3,16 @@
...
- */ 
+ */
+/**
+ * Implements hook_preprocess_HOOK() for HTML document templates.
+ */
+function suiadmin_theme_preprocess_html(&$variables) {
+
+  // Add adminimal class to the body.
+  $variables['attributes']['class'][] = 'suiadmin';
+
```
Comment (egorbogatyrev) [inline suiadmin_theme.theme]: useless empty line

File: suiadmin_theme.theme [hunk 3]
```diff
@@ -23,7 +32,33 @@ function suiadmin_theme_preprocess_field(&$variables) {
...
+/**
+ * Implements hook_preprocess_HOOK() for menu-local-action templates.
+ */
+function suiadmin_theme_preprocess_menu_local_action(array &$variables) {
+  $variables['link']['#options']['attributes']['class'][] = 'ui';
+  $variables['link']['#options']['attributes']['class'][] = 'primary';
+  $variables['link']['#title'] = [
+    '#markup' => '<i class="add square icon"></i> ' . t('Add content'),
+  ];
+}
```
Comment (egorbogatyrev) [inline suiadmin_theme.theme]: add empty line after this

File: suiadmin_theme.theme [hunk 4]
```diff
@@ -3,7 +3,16 @@
 /**
  * @file
  * Functions to support theming in the Semantic UI Admin Theme.
- */ 
+ */
```
Comment (egorbogatyrev) [inline suiadmin_theme.theme]: add empty line after this

Comment (DimDev) [inline suiadmin_theme.theme]: Done

File: suiadmin_theme.theme (hunk 3 above)
Comment (DimDev) [inline suiadmin_theme.theme]: Done

File: css/custom.css (hunk 1 above)
Comment (DimDev) [inline css/custom.css]: Done

**Pull request merged**